Lysov/functions.js

```diff
@@ -0,0 +1,125 @@
+/* #1. Split and Merge function */
+function splitAndMerge (str, sp) {
+  
+  var splitOnWords = str.split(' ');
+ 
+  splitOnWords = splitOnWords.map(function(words) {
+    return words.split('').join(sp);
+  });
+  return splitOnWords.join(' ');
+}
+
+//console.log(splitAndMerge("hello world!", ','));
+
+/* #2. Convert function */
+function convert (obj) {
+  mass = [];
```

Comment:
variables should always be declared using either `var`, `let` or `const`. In case they don't they are assigned to global context. in strict mode this assignment will lead to an error.